src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/ActivityContext.cs

```diff
@@ -59,6 +59,40 @@ public ActivityContext(ActivityTraceId traceId, ActivitySpanId spanId, ActivityT
         /// </remarks>
         public bool IsRemote { get; }
 
+        /// <summary>
+        /// Parse W3C trace parent and state headers to ActivityContext object.
+        /// </summary>
+        /// <param name="traceParent">W3C trace parent header.</param>
+        /// <param name="traceState">Trace state.</param>
```

Comment:
nits: Kind of odd to have W3C in the one comment but not the other. How about...
```csharp
        /// <param name="traceParent">W3C traceparent header value.</param>
        /// <param name="traceState">W3C tracestate header value.</param>
```
Personally not a fan of the casing "traceparent" & "tracestate" but that's how they are named in the [spec](https://www.w3.org/TR/trace-context/#design-overview).

For the method summary, might want to refer to them collectively as Trace Context? Eg:
```csharp
        /// <summary>
        /// Parse W3C Trace Context header values into an ActivityContext object.
        /// </summary>
```

Reply:
> Personally not a fan of the casing "traceparent" & "tracestate" but that's how they are named in the spec.

I prefer to case it as the rest of C# method parameters.

Reply:
Thanks @CodeBlanch I have fixed the comments as you suggested.